finat/ufl/mixedelement.py

```diff
@@ -87,7 +87,7 @@ def reconstruct_from_elements(self, *elements):
             return self
         return MixedElement(*elements)
 
-    def symmetry(self, domain):
```

Comment:
I agree with this change. The symmetry should only be imposed on the reference components